tests/datastore.py

```diff
@@ -29,3 +37,11 @@ def get_url(self, fname):
     base_url='https://data.kitware.com/api/v1/file/hashsum/{algo}/{hashvalue}/download',
     registry=registry,
 )
+
+lowres_datastore = DKCPooch(
```

Comment:
Is there a reason to create a second datastore?  It looks like you enumerate the files you are using anyway.